src/styles/_aboutUs.scss

```diff
@@ -0,0 +1,25 @@
+.aboutUs {
+  margin-bottom: 120px;
```

Comment:
Like for me camel case in CSS, it's not a good idea, but we should ask this question for sure.